src/tree/VirtualMachineTreeItem.ts

```diff
@@ -35,7 +35,7 @@ export class VirtualMachineTreeItem extends AzureTreeItem {
     }
 
     public get description(): string | undefined {
-        return this._state && this._state.toLowerCase().includes('deallocated') ? 'Stopped' : undefined;
+        return this._state !== 'running' ? this._state : undefined;
     }
 
```

Comment:
How come you got rid of `toLowerCase`? I would keep it just in case they ever fix their casing to be more similar to what you like